spotdl/search/spotifyClient.py

```diff
@@ -16,36 +16,49 @@ def __call__(cls):
                             '(client_id, client_secret) first.')
         return cls._instance
 
-    def init(cls, client_id: str, client_secret: str) -> "Singleton":
+    def init(cls, client_id: str, client_secret: str, userAuth: bool) -> "Singleton":
```

Comment:
please use PEP-8 compatible variable names

https://github.com/spotDL/spotify-downloader/pull/1201#discussion_r592337788

```suggestion
    def init(cls, client_id: str, client_secret: str, user_auth: bool) -> "Singleton":
```

Reply:
You should correct the rest of the functions